rewards/classes/RewardsManager.py

```diff
@@ -170,11 +135,23 @@ def calculate_all_sett_rewards(
         all_rewards = []
         for sett in setts:
             token = make_contract(sett, "ERC20", self.chain)
-
             console.log(f"Calculating rewards for {token.name().call()}")
-            all_rewards.append(self.calculate_sett_rewards(sett, all_schedules[sett]))
+            rewards, flat, boosted = self.calculate_sett_rewards(
+                sett, all_schedules[sett]
+            )
+            desc = f"**Boosted Rewards**\n\n{boosted.totals_info(self.chain)}\n\n**Flat Rewards**\n\n{flat.totals_info(self.chain)}"
+            sett_token = fetch_token(self.chain, sett)
+            sett_name = sett_token.get("name", "")
+            send_message_to_discord(
+                f"Rewards for {sett_name}",
+                description=desc,
+                fields=[],
+                username="Rewards Bot",
+                url=self.discord_url,
+            )
+            all_rewards.append(rewards)
 
-        return combine_rewards(all_rewards, self.cycle + 1)
+        return combine_rewards(all_rewards, self.cycle)
```

Comment:
did we mean to get rid of this increment?

Reply:
Yes, because self.cycle is already incremented to the next cycle. This has no affect due to the because the correct value is used when it is finally combined

Reply:
sounds good